src/test/java/estaruma/cupcake/BakeryTest.java

```diff
@@ -0,0 +1,52 @@
+package estaruma.cupcake;
+
+import java.util.List;
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+public class BakeryTest {
```

Comment:
within BakeryTest we should probably be checking name and price for cakes with a combination of toppings 

may be something like the below combination? Could you also try and write a Parameterised test?
About name function or method
The name function should return “🧁”
The name function should return “🍪”
The name function should return “🧁 with 🍫”
The name function should return “🍪 with 🍫”
The name function should return “🍪 with 🍫 and 🥜”
The name function should return “🍪 with 🥜 and 🍫”

About price function or method
The price function should return 1$ for “🧁”
The price function should return 2$ for “🍪”
The price function should return 1.1$ for “🧁 with 🍫”
The price function should return 2.1$ for “🍪 with 🍫”
The price function should return 2.2$ for “🍪 with 🥜”